prisma/schema.prisma

```diff
@@ -8,23 +8,26 @@ datasource db {
 }
 
 model Comment {
-  id          Int       @id @default(autoincrement())
-  accidentId  String    @db.VarChar
-  text        String?
-  authorId    String    @db.VarChar
-  isPublished Boolean   @default(false)
-  createDate  DateTime? @default(now()) @db.Date
-  updateDate  DateTime? @default(now()) @db.Date
-  author      User      @relation(fields: [authorId], references: [id])
+  id          Int      @id @default(autoincrement())
+  accidentId  String   @map("accident_id") @db.VarChar
+  text        String
+  authorId    String   @map("author_id") @db.VarChar
+  isPublished Boolean  @default(false) @map("is_published")
+  createdAt   DateTime @default(now()) @map("created_at")
+  updatedAt   DateTime @default(now()) @map("updated_at")
+  author      User     @relation(fields: [authorId], references: [id])
+
+  @@map("comments")
```

Comment:
Using `@@map` for entities and @map for some fields to end up with `snake_case` in Postgres:

<img width="189" alt="Screenshot 2022-02-05 at 23 35 59" src="https://user-images.githubusercontent.com/608862/152662569-816d0234-b48c-48df-8c83-7402603bea43.png">


This is a standard convention there and will be handy for other clients in other apps (not Prisma)

Reply:
Prev. screenshot is out of date; `author_id` is also `snake_case`:

<img width="195" alt="Screenshot 2022-02-06 at 01 45 58" src="https://user-images.githubusercontent.com/608862/152664940-f759b3c3-29c7-4fe2-bb99-1aea64bc2f28.png">
.